src/BlazorWebView/src/Maui/Android/BlazorWebChromeClient.cs

```diff
@@ -22,5 +27,72 @@ public override bool OnCreateWindow(Android.Webkit.WebView? view, bool isDialog,
 			// We don't actually want to create a new WebView window so we just return false 
 			return false;
 		}
+
+		public override bool OnShowFileChooser(Android.Webkit.WebView? view, IValueCallback? filePathCallback, FileChooserParams? fileChooserParams)
+		{
+			if (filePathCallback is null)
+			{
+				return base.OnShowFileChooser(view, filePathCallback, fileChooserParams);
+			}
+
+			CallFilePickerAsync(filePathCallback, fileChooserParams).FireAndForget();
```

Comment:
Actual logging (via error callback) will be handled via https://github.com/dotnet/maui/issues/4441.